int64 vehicle_id
float64[7] pose
float64[6] twist
float64[6] accel